.github/dependabot.yml

```diff
@@ -2,27 +2,27 @@ version: 2
 updates:
   - package-ecosystem: "cargo"
     directory: "/"
-    labels: ["A0-pleasereview", "A4-insubstantial", "E2-forcemacos"]
+    labels: ["A4-insubstantial", "E2-forcemacos"]
     schedule:
       interval: "weekly"
       day: "friday"
     commit-message:
-      prefix: "[depbot] "
+      prefix: "[dep] "
 
   - package-ecosystem: "cargo"
```

Comment:
Seems this step isn't required because we have unified workspace for main crates and demos